### DISCUSSION TITLE
How do I set `throwIfNamespace: false`

### DISCUSSION THREAD

**Discussion by remy** [vercel/next.js#17697] [Help] [answered]
I'm trying to avoid putting 100s of SVG structures in my DOM, so I want to use `<use>` but getting the React warning `React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning.`.

I just have no idea where to set this - guessing in `.babelrc` I don't know which Next babel module is being used to do the transform.

**Comment by jamesmosier** [comment, marked as answer]
Hi @remy 

I believe you'd want a `.babelrc` that looks like this:

```js
{
  "presets": [
    [
      "next/babel",
      {
        "preset-react": { "throwIfNamespace": false },
      }
    ]
  ],
  "plugins": []
}
```

It looks like [the source](https://github.com/vercel/next.js/blob/7108567b06bba6586296fe2bb7e6957410147c8f/packages/next/build/babel/preset.ts#L124) takes these options and merges them into the `@babel/preset-react` preset.

**Reply by adamk33n3r** [reply]
I have added this but I am still getting the error.
![image](https://user-images.githubusercontent.com/1350444/155861793-a9c3bb93-7044-4f30-8ae1-682008445e73.png)

**Reply by amirho1** [reply]
@adamk33n3r  did you find another  solution I'm facing this problem too I added the ```.babelrc``` but it didn't work

**Reply by adamk33n3r** [reply]
I didn't. I'm ignoring it for the moment because the svgs that were giving me the error aren't necessary for me to use yet, it was just finding them in the directory. So I just movies the ones I wanted into a subdirectory. But I would very much like to know how to solve this. It's so confusing that there are like 3 or 4 levels of configurations while using next.

**Reply by amirho1** [reply]
I found another solution that is like when in React u write onClick u have to change all attributes of SVG file like that for example:  xmlns:sodipodi turn it to xmlnsSodipodi. it's working for me

**Reply by martins-ngene** [reply]
The above code from @jamesmosier works for me. Thank you sir @jamesmosier
